dev/com.ibm.ws.security.jwt/resources/com/ibm/ws/security/jwt/internal/resources/JWTMessages.nlsprops

```diff
@@ -131,13 +128,13 @@ JWT_INVALID_CLAIMS_ERR.useraction=Ensure that the correct set of claims are prov
 
 # Do not translate "JSON Web Token", "JWT"
 # 0=Issuer claim of the token, 1=JWT configuration ID, 2=Trusted issuers specified in server configuration
-JWT_ISSUER_NOT_TRUSTED=CWWKS6022E: The issuer [{0}] of the provided JSON web token (JWT) is not listed as a trusted issuer in the [{1}] JWT configuration. The trusted issuers are [{2}].
+JWT_ISSUER_NOT_TRUSTED=CWWKS6022E: The issuer [{0}] of the provided JSON Web Token (JWT) is not listed as a trusted issuer in the [{1}] JWT configuration. The trusted issuers are [{2}].
 JWT_ISSUER_NOT_TRUSTED.explanation=The server configuration specifies which issuers are trusted when validating JWTs. The provided token did not originate from a trusted issuer, so the token cannot be validated.
 JWT_ISSUER_NOT_TRUSTED.useraction=Obtain a new token from one of the trusted issuers specified in your JWT configuration, or add the issuer specified in the message as a trusted issuer in your JWT configuration.
 
 # Do not translate "JSON Web Token", "JWT"
 # 0=Audience claim of the token, 1=JWT configuration ID, 2=Trusted audiences specified in server configuration
-JWT_AUDIENCE_NOT_TRUSTED=CWWKS6023E: The audience [{0}] of the provided JSON web token (JWT) is not listed as a trusted audience in the [{1}] JWT configuration. The trusted audiences are [{2}].
+JWT_AUDIENCE_NOT_TRUSTED=CWWKS6023E: The audience [{0}] of the provided JSON Web Token (JWT) is not listed as a trusted audience in the [{1}] JWT configuration. The trusted audiences are [{2}].
```

Comment:
Change `audience [{0}]` to `[{0}] audience` unless the variable has to be after the noun.